lib/xboxkrnl/xboxkrnl.h

```diff
@@ -2682,7 +2682,7 @@ XBAPI NTSTATUS NTAPI NtSetEvent
 
 /**
  * Resumes the target thread (see NtSuspendThread) and optionally returns the previous suspend count.
- * @param The handle of the thread object to resume.
```

Comment:
nit: This one has dot at end, others don't.
Not an issue with your change, but something I first noticed here.

Reply:
Just checking @thrimbor, because you pushed but didn't do this: Will this remain as-is?

Eitherway: I reviewed and tested locally. Feel free to self-merge at your discretion. If not merged already I'll probably merge it later today.

Reply:
We seem to have a mix of the comments ending with a dot and those who don't - if we decide that one of those styles is preferred, I think it's best to change them in a separate commit/PR.